tensorflow/python/ops/numpy_ops/np_array_ops.py

```diff
@@ -24,6 +24,7 @@
 import numpy as np
 import six
 
+import tensorflow as tf
```

Comment:
We never `import tensorflow as tf` from within the TensorFlow codebase. We should be importing the files that define the functions we want (looks like you want `from tensorflow.python.framework import tensor_util`), and use them directly like other imports in this file (specifically here `tensor_util.is_tensor`). 

You'll probably want to remove it from the test file as well.